ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -584,35 +584,3 @@ ethernet_interfaces:
     description: Load Interval
     type: switched
     load_interval: 5
-
-  Ethernet50:
-    description: SFlow Interface Testing - SFlow ingress enabled
-    sflow:
-      enable: true
-
-  Ethernet51:
-    description: SFlow Interface Testing - SFlow egress enabled
-    sflow:
-      egress:
-        enable: true
-
-  Ethernet52:
-    description: SFlow Interface Testing - SFlow ingress and egress unmodified enabled
-    sflow:
-      enable: true
-      egress:
-        unmodified_enable: true
-
-  Ethernet53:
-    description: SFlow Interface Testing - SFlow ingress and egress disabled
-    sflow:
-      enable: false
-      egress:
-        enable: false
-
-  Ethernet54:
-    description: SFlow Interface Testing - SFlow ingress and egress unmodified disabled
-    sflow:
-      enable: false
-      egress:
-        unmodified_enable: false
```

Comment:
any reason for deleting the sflow tests here? looks like the PR needs to be rebased to keep the latest changes in place

Reply:
Fixed.